apps/consulting/next.config.js

```diff
@@ -29,6 +29,17 @@ const nextConfig = {
         permanent: false,
         destination: '/careers',
       },
+      {
+        source: '/:path*',
+        has: [
+          {
+            type: 'host',
+            value: '^labs[.-].+$',
```

Comment:
Could you add a line explaining this regex? For example, why are you matching "labs" at the beginning of the string, but then anything at the end?

Reply:
Sure, will do. The terminating `$` is not necessary, I'll remove for simplicity.

Reply:
Ok, comment added, thoughts pls @gabalafou.

Reply:
Looks good, thanks!